pkg/api/v1beta3/types.go

```diff
@@ -814,6 +814,10 @@ type PodSpec struct {
 	// NodeSelector is a selector which must be true for the pod to fit on a node
 	NodeSelector map[string]string `json:"nodeSelector,omitempty" description:"selector which must match a node's labels for the pod to be scheduled on that node"`
 
+	// ServiceAccount is the name of the ServiceAccount to use to run this pod
+	// The pod will be allowed to use secrets referenced by the ServiceAccount
+	ServiceAccount string `json:"serviceAccount" description:"name of the ServiceAccount to use to run this pod"`
```

Comment:
Why just a name here, but a full ObjectReference in the ServiceAccount object? I think just a name is easier for humans to specify, but perhaps harder for generic clients to comprehend.

We should decide what principle we'll use to decide, and try to be consistent in v1:
https://github.com/GoogleCloudPlatform/kubernetes/issues/7018#issuecomment-99630042